pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -113,6 +113,15 @@ type PrebindPlugin interface {
 	Prebind(pc *PluginContext, p *v1.Pod, nodeName string) *Status
 }
 
+// PostbindPlugin is an interface that must be implemented by "postbind" plugins.
+// These plugins are called after a pod was scheduled with a cycle.
+type PostbindPlugin interface {
+	Plugin
+	// Postbind is called after binding a pod. All postbind plugins must return
```

Comment:
Please change to:

Postbind is called after a pod is successfully bound. These plugins are informational. A common application of this extension point is for cleaning up. If a plugin needs to clean-up its state after a pod is scheduled and bound, Postbind is the extension point that it should register.